storage_service/locations/tests/test_duracloud.py

```diff
@@ -308,6 +309,7 @@ def test_move_from_ss_chunked_file(self):
         shutil.copy(os.path.join(FIXTURES_DIR, "chunk_file.txt"), str(self.tmpdir))
         file_path = str(self.tmpdir / "chunk_file.txt")
         self.ds_object.CHUNK_SIZE = 10 * 1024  # Set testing chunk size
+        self.ds_object.BUFFER_SIZE = 1
```

Comment:
If I remember correctly this was needed because the default buffer size is `1MB` and it also affects the chunking functionality in Duracloud.

The files used in the test fixtures are very small (a few KBs) so they didn't really trigger chunking. Setting the buffer size this small guarantees we'll hit that logic.

Reply:
Nice.